src/component/actions/Action/Action.test.ts

```diff
@@ -1,67 +1,44 @@
 import { getTypeConfig } from '../../../config';
-import { createSetup, getSpecWrapper } from '../../../testUtils';
-import { getLanguage, getLocalization } from '../../../utils';
 import {
+    composeComponentTestUtils,
+    configProviderContext,
+    genExit,
+    genFlowNode,
+    genSendMsgAction,
+    genStartFlowAction,
+    genStartFlowNode,
+    setMock,
+    getSpecWrapper
+} from '../../../testUtils';
+import { getLanguage, getLocalization, setFalse, setTrue, set } from '../../../utils';
+import {
+    actionBodySpecId,
     actionContainerSpecId,
+    actionInteractiveDivSpecId,
     actionOverlaySpecId,
     ActionWrapper,
-    ActionWrapperProps,
-    actionInteractiveDivSpecId,
-    actionBodySpecId
+    ActionWrapperProps
 } from './Action';
 import { FlowNode, SendMsg, StartFlow, SwitchRouter } from '../../../flowTypes';
 
-const config = require('../../../../__test__/assets/config');
-
-const sendMsgAction: SendMsg = {
-    uuid: 'send_msg_action0',
-    type: 'send_msg',
-    text: 'Hello World!'
-};
-
-const sendMsgAction1: SendMsg = {
-    uuid: 'send_msg_action1',
-    type: 'send_msg',
-    text: 'Hello World!'
-};
-
-const sendMsgNode: FlowNode = {
-    uuid: 'send_msg_node',
-    actions: [sendMsgAction],
-    exits: []
-};
-
-const startFlowAction: StartFlow = {
-    uuid: 'start_flow_action',
-    type: 'start_flow',
-    flow_name: 'Flow to Start',
-    flow_uuid: 'flow_to_start'
-};
-
-const startFlowNode: FlowNode = {
-    uuid: 'start_flow_node',
-    actions: [startFlowAction],
-    exits: [],
-    router: {
-        type: 'switch',
-        operand: '@child'
-    } as SwitchRouter,
-    wait: {
-        type: 'flow',
-        flow_uuid: 'flow_to_start'
+const sendMsgAction = genSendMsgAction();
+const sendMsgAction1 = genSendMsgAction({ uuid: 'send_msg-1', text: 'Yo!' });
+const sendMsgNode = genFlowNode({ actions: [sendMsgAction], exits: [genExit()] });
+const startFlowAction = genStartFlowAction();
+const startFlowNode = genStartFlowNode(startFlowAction);
```

Comment:
This makes a lot of sense and is pretty DRY. Especially if we end up changing the flow format, this will be much easier to maintain. Can we call them `createBlah` instead of `gen`? Maybe they alos deserve their own file in the test utils.

Reply:
Totally, moved these to `assetCreators` and gave them the 'create' prefix here https://github.com/nyaruka/floweditor/pull/270/commits/d86214d4fb59dfa9385c9917c57c03ec0d2f453f